features/wijzigingen.feature

```diff
@@ -0,0 +1,71 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik wijzigingen doorkrijgen op gebruikers die ik volg
```

Comment:
Ook hier klopt de titel niet. Je krijgt geen wijzigingen door van een gebruiker. Je krijgt alleen de bsn van personen waarvoor je een volgindicatie hebt geplaatst en die wijzigingen heeft vanaf een bepaalde datum

Reply:
Het is mij niet duidelijk wat het gedrag is/moet zijn als ik geen vanaf datum opgeef. Dat staat ook niet in de OAS specs. Ik concludeer uit wat ik lees dat de datum waarop ik de volgindicatie heb geplaatst wordt gebruikt als vanaf datum. Klopt dit?

Wat ik ook mis is een scenario voor meerdere wijzigingen aan één persoon vanaf een datum. Krijg ik de bsn één keer? Of voor elke wijziging één?

Reply:
ook deze feature beschrijft voor de implementatie van de update applicatie hoe deze moet werken. de doelgroep is dus niet de developer van een client, maar de developer van de provider. Deze heeft naast de update API nog twee bronnen: binnengemeentelijke wijzigingen (burgerzakenapplicatie) en buitengemeentelijke wijzigingen (GBA-V / VOA).

Reply:
ik voeg wat toelichting bovenaan de feature toe over doel van de feature en de algemene werking

ik voeg ook een scenario toe met meerdere wijzigingen, waarbij de gebruiker het burgerservicenummer maar één keer krijgt

Reply:
> `Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen` begrijp ik niet wat de invloed is van de datum waarop ik de wijzigingen vraag

Hier heb ik de datum alleen toegevoegd om de sequentie van gebeurtenissen/handelingen expliciet te maken en te illustreren dat de wijzigingen worden gevraagd op een latere datum (of in een ander scenario op dezelfde datum) als de wijziging.